src/test/java/no/unit/nva/model/contexttypes/BookTest.java

```diff
@@ -29,11 +26,10 @@ void objectMapperReturnsBookWhenInputIsValidJson() throws JsonProcessingExceptio
         String expectedTitle = "The history of JSON in this project";
         Level expectedLevel = Level.LEVEL_2;
         String expectedIsbn = "123 123123 1112";
-        String json = generateJsonFromTemplate(expectedType, expectedTitle, null, expectedLevel,
+        String json = generateJsonFromTemplate(null, expectedLevel,
                 true, true, expectedIsbn);
         Book book = objectMapper.readValue(json, Book.class);
-        assertEquals(expectedTitle, book.getTitle());
-        assertEquals(NullSeries.class, book.getSeries().getClass());
+        assertNull(book.getSeriesTitle());
```

Comment:
It would be better if you created a book with the same attributes and then checked the equality on book.
`assertEquals(actualBook,expectedBook)